torch/distributed/pipelining/schedules.py

```diff
@@ -2213,7 +2295,7 @@ def _ready_to_schedule(action: Optional[_Action]) -> bool:
                 ):
                     return True
             return False
-        elif action.computation_type == B:
+        elif action.computation_type in (BACKWARD_INPUT, FULL_BACKWARD):
```

Comment:
I think I should change this to just 'FULL_BACKWARD' to be consistent with the rest of this PR and then add BACKWARD_INPUT back in in the later PR where I fix other inconsistencies.